### DISCUSSION TITLE
Caching

### DISCUSSION THREAD

**Discussion by SerjNikitin** [grafana/grafana-json-datasource#327] [Q&A]
Hello, what is the cache key made of? Do I understand correctly that the variables from the request body do not get into it?
<img width="786" alt="image" src="https://user-images.githubusercontent.com/76908941/213913418-8b9757ef-da4c-424b-8b1a-44fa208dd99b.png">

**Comment by yesoreyeram** [comment]
you can find the cache key here https://github.com/grafana/grafana-json-datasource/blob/cb9c3ce7fa7055b4429a0bef660736a0bdc6c37e/src/api.ts#L77

basically it is url and doesn't include body.

**Reply by SerjNikitin** [reply]
thanks)
